aspnetcore/tutorials/grpc/grpc-start/samples/GrpcStart/GrpcGreeter.Server/Program.cs

```diff
@@ -16,10 +17,15 @@ public static void Main(string[] args)
         }
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
-                {
-                    webBuilder.UseStartup<Startup>();
-                });
+             Host.CreateDefaultBuilder(args)
```

Comment:
What's up with the spacing here?

Reply:
That gets rid of the horizontal scroll bar on tablets.